apps/dav/lib/CardDAV/AddressBookImpl.php

```diff
@@ -107,6 +107,7 @@ public function getDisplayName() {
 	 * 	- 'escape_like_param' - If set to false wildcards _ and % are not escaped
 	 * 	- 'limit' - Set a numeric limit for the search results
 	 * 	- 'offset' - Set the offset for the limited search results
+	 * 	- 'wildcard' - Whether the search should use wildcards
```

Comment:
```suggestion
	 * 	- 'wildcard' - Whether the search should use wildcards
	 * @psalm-param ?array{types: ?bool, escape_like_param: ?bool, limit: ?int, offset: ?int, wildcard: ?string} $options
```

Unrelated improvement that could be added at the same time for better static analysis :) The same should be added to the other method definitions.

Reply:
Added